src/recipes/banana-smoothie.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Banana Smoothie",
+    "description": "Enjoy a Banana Smoothie. This recipe contains raw eggs.",
+    "github": "kevinmel2000",
+    "ingredients": [
+        "1 banana",
+        "1 cup milk",
+        "1 teaspoon vanilla extract",
+		"1 egg",
+		"2 tablespoons white sugar",
+		"1 pinch ground cinnamon"
+    ],
+    "directions": [
+        "In a blender, combine banana, milk, vanilla, egg and sugar. Blend until smooth. Pour into a tall glass and top with a pinch of cinnamon. ",
+        "5 Minute"
```

Comment:
Sorry, one more small item. Should there be more to this line `5 Minute`?

Reply:
Ok , I juts commit the change Direction Fix